src/lightning_app/cli/cmd_apps.py

```diff
@@ -0,0 +1,81 @@
+import json
+from datetime import datetime
+
+from lightning_cloud.openapi import Externalv1LightningappInstance, V1LightningappInstanceState
+from rich.console import Console
+from rich.table import Table
+from rich.text import Text
+
+from lightning_app.cli.core import Formatable
+from lightning_app.utilities.cloud import _get_project
+from lightning_app.utilities.network import LightningClient
+
+
+class AppManager:
+    """AppManager implements API calls specific to Lightning AI BYOC apps."""
+
+    def __init__(self):
+        self.api_client = LightningClient()
+
+    def list(self, cluster_id=None):
+        project = _get_project(self.api_client)
+
+        args = {
+            "project_id": project.project_id,
+        }
+        if cluster_id is not None:
+            args["cluster_id"] = cluster_id
+
+        resp = self.api_client.lightningapp_instance_service_list_lightningapp_instances(**args)
+        apps = resp.lightningapps
+        while resp.next_page_token is not None and resp.next_page_token != "":
```

Comment:
nit: we could probably use a higher limit as well here (regardless, both are not apparently implemented in the BE)

Reply:
are you talking about the page size?

Reply:
yes!

Reply:
made explicit and bumbed to 100 for now :+1: